packages/core/src/accordion/AccordionHeader.css

```diff
@@ -35,11 +35,10 @@
 
 .saltAccordionHeader .saltAccordionHeader-icon {
   height: var(--salt-size-base);
-  transition: transform var(--salt-duration-perceptible) ease-in-out;
 }
 
 .saltAccordionHeader[aria-expanded="true"] > .saltAccordionHeader-icon {
-  transform: rotate(90deg);
+  transform: rotate(180deg);
```

Comment:
Now without the animation, and with @pseys semantic icon naming, worth to check whether we should actually use 2 different icons to achieve 2 states, instead of doing transform.